src/codeu/chat/client/simplegui/ChatSimpleGui.java

```diff
@@ -62,56 +62,75 @@ private Border paneBorder() {
   private void initialize() {
 
     // Outermost frame.
-    // NOTE: may have tweak size, or place in scrollable panel.
-    mainFrame = new JFrame("Chat");
+    // NOTE: may have to tweak size, or place in scrollable panel.
+    mainFrame = new JFrame("ChatU");
     mainFrame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
     mainFrame.setSize(790, 450);
 
     // Main View - outermost graphics panel.
     final JPanel mainViewPanel = new JPanel(new GridBagLayout());
-    mainViewPanel.setBorder(paneBorder());
+    //mainViewPanel.setBorder(paneBorder());
 
-    // Build main panels - Users, Conversations, Messages.
-    final JPanel usersViewPanel = new UserPanel(clientContext);
-    usersViewPanel.setBorder(paneBorder());
-    final GridBagConstraints usersViewC = new GridBagConstraints();
+    // Build main panels - Sign In, Create Account.
+
+    // Panel with just label -- Cleaner to make a new class?
+    final JPanel labelPanel = new JPanel();
+    labelPanel.setLayout(new BoxLayout(labelPanel, BoxLayout.Y_AXIS));
+    labelPanel.setBorder(paneBorder());
+    labelPanel.add(new JLabel("Welcome to ChatU!"));
+    labelPanel.add(new JLabel("Please choose from an option below:"));
+    final GridBagConstraints labelViewC = new GridBagConstraints();
 
-    final MessagePanel messagesViewPanel = new MessagePanel(clientContext);
-    messagesViewPanel.setBorder(paneBorder());
-    final GridBagConstraints messagesViewC = new GridBagConstraints();
+    final JPanel userSignIn = new SignInPanel();
+    userSignIn.setBorder(paneBorder());
+    final GridBagConstraints usersViewC = new GridBagConstraints();
```

Comment:
what is the C in userViewC?  Prefer to have descriptive names even if they're long (i.e. userViewConstraints?)